airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1962,18 +1962,22 @@ components:
           default: true
         slackConfiguration:
           $ref: "#/components/schemas/SlackNotificationConfiguration"
+        emailConfiguration:
+          $ref: "#/components/schemas/EmailNotificationConfiguration"
     SlackNotificationConfiguration:
       type: object
       required:
         - webhook
       properties:
         webhook:
           type: string
+    EmailNotificationConfiguration:
+      type: object
```

Comment:
Should there be some enum settings on the email provider?

For the moment it's serviced by customer.io so we should probably make that clear in the config? Maybe we'll have a different email service later on?

Reply:
Actually, customer.io can be used to send notifications in sms/slack/etc other than email, does this notification via customer.io enable someone to configure their account to trigger other things than email? 

So maybe naming it email notification is not correct?

Reply:
Yeah, I struggled over naming this a generic email notification client vs customerio notification client. I didn't have a good reason other than, this is the only way we can notify through email for now, so I'm happy to rename this to a customer.io specific implementation.

Right now this notification client can only trigger emails through customer.io, but i don't see why it shouldn't be re-used in the future if we want to use customer.io to send sms/other things as well.

I'll go ahead and rename everything to be customer.io specific instead of email.